crypto/s2n_pq.c

```diff
@@ -33,3 +33,15 @@ bool s2n_pq_is_enabled()
 {
     return s2n_libcrypto_supports_evp_kem();
 }
+
+bool s2n_libcrypto_supports_mlkem()
+{
+    /* S2N_LIBCRYPTO_SUPPORTS_MLKEM will be auto-detected and #defined if
+     * ./tests/features/S2N_LIBCRYPTO_SUPPORTS_MLKEM.c successfully compiles
+     */
```

Comment:
I'm not seeing the feature test for this. Is it already committed and I'm missing it? Or did you forget to commit it?

If it is missing, that may be a sign we need a unit test to make sure it's true in at least one environment where we know it should be true.

Reply:
Fixed